next.config.mjs

```diff
@@ -46,7 +46,7 @@ const nextConfig = {
 
     return config
   },
-  // TODO: temporary patch. investigate
+  turbopack: {},
```

Comment:
This is not actually needed to run it in turbopack mode.
If we are not going to populate the config, it is probably better that we are reminded to do that every time we run it.

So I'd suggest removing this empty config.

Reply:
it throws errors or notifications if you completely remove the config.